ecommerce/models.py

```diff
@@ -10,7 +10,7 @@ def __str__(self):
 
 class Product(models.Model):
     product_name = models.CharField(max_length=200)
-    image = models.ImageField(default='/images/product_pics/default.png', upload_to='images/product_pics')
+    image = models.ImageField(default='/images/product_pics/default.png', upload_to='ecommerce/media/images/product_pics')
```

Comment:
1. Tạo folder media ngang hàng với file manage.py
2. upload image có tên default.png lên folder media/product_pics/
3. Edit lại model nha
```suggestion
    image = models.ImageField(default='/product_pics/default.png', upload_to='product_pics')
```